openshift-test-utils/src/main/java/io/openshift/booster/test/OpenShiftTestAssistant.java

```diff
@@ -176,4 +176,29 @@ public void awaitPodReadinessOrFail(Predicate<Pod> filter) {
                                                   }
         );
     }
+
+    public void scale(final int replicas) {
+        System.out.println(String.format("Scaling replicas from %s to %s.", deploymentConfig().getStatus()
+                .getAvailableReplicas(), replicas));
+
+        this.client
+                .deploymentConfigs()
+                .inNamespace(this.project)
+                .withName(this.applicationName)
+                .scale(replicas);
+
+        await().atMost(5, TimeUnit.MINUTES)
+                .until(() -> deploymentConfig()
+                        .getStatus()
+                        .getAvailableReplicas() == replicas
```

Comment:
I have updated the code to be compatible with OCP 3.4. Left `deploymentConfig()` method there as it might be usefull for somebody, otherwise I will delete it. How do we get this change to e.g. configmap boosters? Releasing new booster-common + parent and then update parent version in a configmap booster?